asab/library/providers/zookeeper.py

```diff
@@ -301,6 +302,11 @@ def build_path(self, path):
 		else:
 			node_path = self.BasePath
 
+		tenant = tenant_var.get()
+
+		if tenant not in [None, ""]:
+			node_path = self.BasePath + "/.tenants/" + tenant + path
```

Comment:
Yo need to do two "hits" to zookeeper - first into `/.tenants/....` and if the item doesn't exist there, then you do hit the "normal" path.